src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/Interop/SafeMsQuicConfigurationHandle.cs

```diff
@@ -156,9 +156,13 @@ private static unsafe SafeMsQuicConfigurationHandle Create(QuicOptions options,
                 ulong ms = (ulong)options.IdleTimeout.Ticks / TimeSpan.TicksPerMillisecond;
                 if (ms > (1ul << 62) - 1) throw new Exception("IdleTimeout is too large (max 2^62-1 milliseconds)");
 
-                settings.IsSetFlags |= QuicSettingsIsSetFlags.IdleTimeoutMs;
                 settings.IdleTimeoutMs = (ulong)options.IdleTimeout.TotalMilliseconds;
             }
+            else
+            {
+                settings.IdleTimeoutMs = 0;
```

Comment:
msquic itself uses 0 to mean infinite?

Reply:
This is what I understand from #55984 explanation.

Reply:
Yep, 0 mean infinite: https://github.com/microsoft/msquic/blob/4b41f2469b68d2fea0682d6f68854399a239b5bf/docs/api/QUIC_SETTINGS.md#members
> How long a connection can go idle before it is gracefully shut down. 0 to disable timeout.

Reply:
Thanks.  Out of curiosity, how do you say "don't let connections go idle at all"?  I'd have expected that to be what 0 meant.  Is there a separate "disable pooling entirely" setting or something, or is that just not a thing at all?

Reply:
I'm not sure if I understand what you mean by "don't let connections go idle at all". Do you mean something like keep alive?

Reply:
> EDIT: Ah, this isn't about pooling, it's about how long a connection in use can go without data being transferred?

yep, it's basically auto-close after a time.